addon/templates/components/from-elsewhere.hbs

```diff
@@ -1,9 +1,9 @@
 {{#if initialized}}
   {{#if hasBlock}}
-    {{yield (get (get service.actives name) 'lastObject.component') }}
+    {{yield (get (get service.actives name) 'lastObject.component') (get (get service.actives name) 'lastObject.params')}}
```

Comment:
Does `get` do anything special in these examples, or could we make these into Computed Properties?

Reply:
The inner `get` is definitely needed because the property itself is dynamic. Doing that in a computed property is very awkward.

The outer `get` wouldn't be necessary if we could say:

```js
(get service.actives name).lastObject.component
```

But that is not valid handlebars. So the outer get is needed, unless we refactor to use `let`.